docs/Infrastructure.md

```diff
@@ -0,0 +1,80 @@
+# Infrastructure
+The Urlist application is a relativly simple application that uses a robust and scalable infrastructure.  The following demonstrates how using Azure services include Front door, API Management and other PaaS offerings we can quickly and easily deploy a global footprint with fast edge access and global data replication using Cosmos DB.
+
+## Azure Infrastruture
+![Infrastucture Diagram](images/infrastructure-diagram.png)
+
+## Naming Conventions
+All resources follow a naming convention that ensures unique naming that includes the following components:
+- Resource Type
+- Business Unit
+- Application Name
+- Environment (dev | test | prod | etc)
+- Region / Location (Multi-region services use **gbl** indicator)
+
+> `${ResourceType}-${BusinessUnit}-${AppName}-${Env}-${Region}`
+### Examples
+- fd-prodoh-urlist-prod-gbl **(Front Door)**
+- rg-prodoh-urlist-prod-westus **(Resource Group)**
+- apim-prodoh-urlist-prod-westus **(API Management)**
+
+## Azure FrontDoor
+Azure Frontdoor is used to provide a central frontdoor and mesh to all micro-services required for the Urlist application.  It consolidates all url endpoints to a single HTTP entrypoint.  Robust routing rules allow routing traffic to the correct microservice based on request paths.
+
+### Features
+- Intelligent routing based on geo-location and latency (edge delivery)
+- Single HTTP entry point (ex: www.urlist.com)
+- Handles TLS offloading for HTTPS (allows underlying services to use standard HTTP)
+- Caching for static content
+
+### Backend Pools
+Two backends are configured for the Urlist application. Backend pools can be configured with different priority or weights based on your business needs.  Each pool also must include a HTTP/HTTPS probe endpoint to monitor a regions health status.  Unhealthy nodes are automatically disabled from the pool.

```

Comment:
We have more than 2 backends configured. I think we have configured it in westus, eastus and centralus.